packages/connect/src/Detector.ts

```diff
@@ -59,6 +60,14 @@ export class Detector {
     return this.#name;
   }
 
+  /**
+   * Returns a boolean showing if the app that was created is interacting with 
+   * an extension or not
```

Comment:
```
Returns `true` if the user has the extension installed otherwise false

@remarks

You should check this if you wish to show UI to encourage users to download and install the extension or if you wish your app only to work with the extension installed.
```